src/Upgrade/Application/Strategy/Composer/Fixer/FeatureDevMasterPackageFixerStep.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\Composer\Fixer;
+
+use Upgrade\Application\Adapter\PackageManagerAdapterInterface;
+use Upgrade\Application\Dto\StepsResponseDto;
+use Upgrade\Domain\Entity\Collection\PackageCollection;
+use Upgrade\Domain\Entity\Package;
+
+class FeatureDevMasterPackageFixerStep extends AbstractFeaturePackageFixerStep
+{
+    /**
+     * @var string
+     */
+    public const ALIAS_DEV_MASTER = 'dev-master as 202000.0';
```

Comment:
I have a proposition - let's make an assumption that the last product release happens in November. In this case we can at least calculate the year correct here: 

$version = date('m') <= 11 ? date('Y') . '00.0' : date('Y', strtotime('+1 year')) . '00.0';

Reply:
I'll change but it doesn't make sense. you don't know next tag